src/Dialect/ONNX/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ add_dependencies(OMONNXOps OMONNXOpsIncGen)
 # Linking dependencies:
 add_dependencies(OMONNXOps
         OMPromotableConstOperandsOpInterface
+        OMResultTypeInferenceInterface
```

Comment:
If the interface is an op interface, then this target name should end with `OpInterface`.